solutions/usaco-553.mdx

```diff
@@ -0,0 +1,98 @@
+---
+id: usaco-553
+source: USACO Gold 2015 Open
+title: Palindromic Paths
+author: Maggie Liu
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_palpath_gold.html)
+
+## Implementation
```

Comment:
move inside cppsection and make it c++ imple

Reply:
this is the template on the guide, so i think this is fine